core/circuit/circuit_test.go

```diff
@@ -32,12 +33,14 @@ func circuitTests() []circuitTest {
 		{0, 1, 0, 2, 1, 1}, // -
 
 		// circuit 1 overloaded
-		{0, 2, 0, 0, 0, 0}, // =
-		{0, 2, 0, 0, 1, 0}, // +
-		{0, 2, 0, 1, 1, 0}, // =
-		{0, 2, 0, 2, 2, 1}, // =
-		{0, 2, 0, 2, 3, 1}, // +
-		{0, 2, 0, 2, 1, 1}, // -
+		{0, 2, 0, 0, 0, 0},   // =
+		{0, 2, 0, 0, 1, 0},   // +
+		{0, 2, 0, 1, 1, 0},   // =
+		{0, 2, 0, 2, 2, 1},   // =
+		{0, 2, 0, 2, 3, 1},   // +
+		{0, 2, 0, 2, 1, 1},   // -
+		{0, 1.1, 0, 2, 1, 1}, // -
```

Comment:
```suggestion

		{0, 1.1, 0, 2, 1, 1}, // -
```

Einfach nur fürs diff